feat: add min versions for mobile SDKs (#23)
amplitude/amplitude-dev-center: docs/data/sdks/ios-swift/index.md

--8<-- "includes/size/ios.md"
    `./measure_cocoapod_size.py --cocoapods AmplitudeSwift:0.7.0`.

!!! info "Minimum versions"

>     Refer to [`Package.swift`](https://github.com/amplitude/Amplitude-Swift/blob/main/Package.swift#L8-L12) for minimum versions for platforms supported by the package

Reviewer: ```suggestion
    For the minimum supported platform versions of this package, see [`Package.swift`](https://github.com/amplitude/Amplitude-Swift/blob/main/Package.swift#L8-L12) on GitHub.
```
